### PR TITLE
AG-9324 Put x-label on top in vertical bullet series

### PR DESCRIPTION
**Pull request by olegat**
https://ag-grid.atlassian.net/browse/AG-9324

**Preview**
Screenshot 2023-11-08 at 12 18 43

### CODE REVIEW

File: packages/ag-charts-community/src/chart/factory/seriesTypes.ts [hunk 1]
```diff
@@ -17,8 +20,9 @@ const STACKABLE_SERIES_TYPES = new Set<SeriesOptionsTypes['type']>();
 const GROUPABLE_SERIES_TYPES = new Set<SeriesOptionsTypes['type']>();
 const STACKED_BY_DEFAULT_SERIES_TYPES = new Set<string>();
 const SWAP_DEFAULT_AXES_CONDITIONS: Record<string, (opts: unknown) => boolean> = {};
+const CUSTOM_DEFAULT_AXES_SWAPPERS: Record<string, SwapperFunction> = {};
 
-export function registerSeries(
+export function registerSeries( //<TOpts extends SeriesOptionsUnion>(
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/factory/seriesTypes.ts]: Nit: Is the comment redundant?

File: packages/ag-charts-enterprise/src/series/bullet/bulletModule.ts [hunk 2]
```diff
@@ -24,6 +24,21 @@ export const BulletModule: _ModuleSupport.SeriesModule<'bullet'> = {
             },
         ],
     },
-    swapDefaultAxesCondition: (series) => series?.direction !== 'horizontal',
+    swapDefaultAxesCondition: (opts) => opts?.direction !== 'horizontal',
+    customDefaultAxesSwapper: (opts) => {
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/series/bullet/bulletModule.ts:28]: I'm a bit confused why we need this extra level of control vs. the result of `swapDefaultAxesCondition()` - isn't the only different the order of declaration of the axes?

**Review by alantreadway — APPROVED** [review]
LGTM other than one nit.

File: packages/ag-charts-community/src/chart/factory/seriesTypes.ts (hunk 1 above)
Comment (olegat) [reply packages/ag-charts-community/src/chart/factory/seriesTypes.ts]: Oh... I thought that I had removed this 🙈 Well spotted! Thanks

**Pull request merged**